packages/paste-website/src/components/breadcrumb/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Link} from 'gatsby';
+
+const CrumbSlashStyled = styled.aside(props => ({
+  padding: `0 ${themeGet('space.space20')(props)}`,
+  display: 'inline',
+  color: themeGet('textColors.colorTextWeak')(props),
+}));
+
+const CrumbSlash: React.FC<{}> = () => <CrumbSlashStyled>/</CrumbSlashStyled>;
+
+const Breadcrumb: React.FC<{}> = ({children}) => {
+  return (
+    <>
+      {React.Children.map(children, child => (
+        <>
+          {child}
+          <CrumbSlash />
+        </>
+      ))}
+    </>
+  );
+};
+
+const BreadcrumbItem = styled(Link)`
+  height: 24px;
+  font-size: 14px;
+  line-height: 1.71;
```

Comment:
Can you write a TODO here to convert these to tokens? line-heights are coming and font-size should already cover it.

Also is the height going to cause a problem if you bump the base font size for the page? Will the font scale (when using rems from tokens) and the fixed height cause issues?